src/main/java/PrefixTree.java

```diff
@@ -0,0 +1,100 @@
+import java.util.ArrayList;
```

Comment:
This file has no place here. As I mentioned previously, please self-review before placing a PR. You are wasting the reviewer's time.